components/dialog/theme.css

```diff
@@ -16,7 +16,7 @@
   position: fixed;
   top: 0;
   width: 100vw;
-  z-index: var(--z-index-higher);
+  z-index: 6000;
```

Comment:
We really need to find a way to handle z-indexes better...

Reply:
I don't know, but we should at least think about it 😄 /cc @nickwaelkens

Reply:
Sigh. Very true. 😢 
https://teamleader.atlassian.net/browse/GFE-6

Reply:
Maybe we can work with more specific names for the different tiers of z-index and specify their use more. Explained here: https://css-tricks.com/handling-z-index/

Reply:
Thing is, how are we going to scale this over multiple apps? I think there needs to be some sort of communication across said apps... Food for thought 👍